src/lib/viewers/controls/media/DurationLabels.scss

```diff
@@ -3,8 +3,19 @@
 .bp-DurationLabels {
     display: flex;
     align-items: center;
+    justify-content: space-between;
     margin-right: 5px;
     margin-left: 5px;
+
+    .bp-DurationLabels-label {
+        color: rgba(255, 255, 255, .8);
+        font-weight: normal;
+        font-size: 14px;
+        font-family: Lato, Arial, sans-serif;
+        font-style: normal;
+        line-height: 20px;
+        letter-spacing: .3px;
```

Comment:
this and font family seem very specific, should this not be default?

Reply:
Came directly form figma.